README.md

```diff
@@ -367,12 +367,12 @@ This is a collection of conference papers that interest me. The emphasis is focu
 - **Gist: Efficient Data Encoding for Deep Neural Network Training.** (Michigan, Microsoft, Toronto)
 - **The Dark Side of DNN Pruning.** (UPC)
 - **Neural Cache: Bit-Serial In-Cache Acceleration of Deep Neural Networks.** (Michigan)
-- EVA^2: Exploiting Temporal Redundancy in Live Computer Vision. (Cornell)
-- Euphrates: Algorithm-SoC Co-Design for Low-Power Mobile Continuous Vision. (Rochester, Georgia Tech, ARM)
-- Feature-Driven and Spatially Folded Digital Neurons for Efficient Spiking Neural Network Simulations. (POSTECH/Berkeley, Seoul National)
-- Space-Time Algebra: A Model for Neocortical Computation. (Wisconsin)
-- Scaling Datacenter Accelerators With Compute-Reuse Architectures. (Princeton)
-- Enabling Scientific Computing on Memristive Accelerators. (Rochester)
+- **EVA^2: Exploiting Temporal Redundancy in Live Computer Vision.** (Cornell)
+- **Euphrates: Algorithm-SoC Co-Design for Low-Power Mobile Continuous Vision.** (Rochester, Georgia Tech, ARM)
+- **Feature-Driven and Spatially Folded Digital Neurons for Efficient Spiking Neural Network Simulations.** (POSTECH/Berkeley, Seoul National)
+- **Space-Time Algebra: A Model for Neocortical Computation.** (Wisconsin)
```

Comment:
Thanks, Ken. I prefer unbold Line 373~375, as the bold font in this survey usually means they are quite related to deep learning accelerators, though I think the three papers are interesting.